### ISSUE TITLE
fatal EPUB build error in daps 1.1.7 [sf#199]

### ISSUE THREAD

**Issue by tomschr** [openSUSE/daps#198]
**Reported by fsundermeyer on 2013-11-22 09:59 UTC**
The following error does not happen with DAPS2

Sources are from
https://github.com/openSUSE/open-build-service-documentation.git

```
daps -d DC-obs-reference-guide --debug epub

##>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
xsltproc --xinclude --output /local/svn/open-build-service-documentation/build/.tmp/epub_obs-reference-guide.xml  /usr/share/daps/daps-xslt/epub/db2db.xsl \
/local/svn/open-build-service-documentation/build/.profiled/opensuse-novell_/book-obs-reference-guide.xml
##<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
WARNING: wrong metadata date format. It must be in one of these forms: YYYY, YYYY-MM, or YYYY-MM-DD. Using current date.
runtime error: file file:///usr/share/xml/docbook/stylesheet/nwalsh/1.78.0/common/common.xsl line 926 element choose
Variable 'stylesheet.result.type' has not been declared.
xmlXPathCompiledEval: 2 objects left on the stack.
/usr/share/daps/make/common.mk:1982: *** [/local/svn/open-build-service-documentation/build/.tmp/epub_obs-reference-guide.xml] Error 10
```

**Comment by tomschr** [comment]
**Commented by sknorr on 2013-11-27 16:34**
- Description has changed:

Diff:

~~~~

--- old
+++ new
@@ -3,16 +3,14 @@
 Sources are from
 https://github.com/openSUSE/open-build-service-documentation.git
 
-DAPS call:
-daps -d DC-obs-reference-guide --debug epub
+    daps -d DC-obs-reference-guide --debug epub
 
-
-##>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
-xsltproc --xinclude --output /local/svn/open-build-service-documentation/build/.tmp/epub_obs-reference-guide.xml  /usr/share/daps/daps-xslt/epub/db2db.xsl \
-  /local/svn/open-build-service-documentation/build/.profiled/opensuse-novell_/book-obs-reference-guide.xml
-##<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
-WARNING: wrong metadata date format. It must be in one of these forms: YYYY, YYYY-MM, or YYYY-MM-DD. Using current date.
-runtime error: file file:///usr/share/xml/docbook/stylesheet/nwalsh/1.78.0/common/common.xsl line 926 element choose
-Variable 'stylesheet.result.type' has not been declared.
-xmlXPathCompiledEval: 2 objects left on the stack.
-/usr/share/daps/make/common.mk:1982: *** [/local/svn/open-build-service-documentation/build/.tmp/epub_obs-reference-guide.xml] Error 10
+    ##>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+    xsltproc --xinclude --output /local/svn/open-build-service-documentation/build/.tmp/epub_obs-reference-guide.xml  /usr/share/daps/daps-xslt/epub/db2db.xsl \
+    /local/svn/open-build-service-documentation/build/.profiled/opensuse-novell_/book-obs-reference-guide.xml
+    ##<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
+    WARNING: wrong metadata date format. It must be in one of these forms: YYYY, YYYY-MM, or YYYY-MM-DD. Using current date.
+    runtime error: file file:///usr/share/xml/docbook/stylesheet/nwalsh/1.78.0/common/common.xsl line 926 element choose
+    Variable 'stylesheet.result.type' has not been declared.
+    xmlXPathCompiledEval: 2 objects left on the stack.
+    /usr/share/daps/make/common.mk:1982: *** [/local/svn/open-build-service-documentation/build/.tmp/epub_obs-reference-guide.xml] Error 10

~~~~

**Comment by tomschr** [comment]
**Commented by sknorr on 2013-11-27 16:34**
Add 4 spaces as indentation to output.

**Comment by fsundermeyer** [comment]
Seems to have been fixed with 1.1.8 or 1.1.9.
